### DISCUSSION TITLE
submit后如果校验不通过会报错，Uncaught (in promise)

### DISCUSSION THREAD

**Discussion by loongcheung** [alibaba/formily#2597] [Q&A 2.x] [answered]
![image](https://user-images.githubusercontent.com/25630765/145321366-af53f97e-f142-4155-9ef0-f16c8bea6068.png)
这是否正常的，官方例子也是如此，https://codesandbox.io/s/vqtf7?file=/App.tsx

**Comment by janryWang** [comment, marked as answer]
http://antd.formilyjs.org/components/submit#api 可以通过onSubmitFailed来catch错误

**Reply by xiaoxixi6633** [reply]
`function submitFailed(feedbacks: IFormFeedback[]) {
  try {
    console.log('feedbacks', feedbacks)
  } catch (error) {}
}`

**Reply by beita-wang** [reply]
请教一下 如果是onInput 方式校验失败，怎么catch错误。
